client/tests/integration/tx_chain_id.rs

```diff
@@ -0,0 +1,74 @@
+use std::str::FromStr;
+
+use iroha_crypto::KeyPair;
+use iroha_data_model::prelude::*;
+use test_network::*;
+
+#[test]
+fn send_tx_with_different_chain_id() {
+    let (_rt, _peer, test_client) = <PeerBuilder>::new().with_port(11_240).start_with_runtime();
+    wait_for_genesis_committed(&[test_client.clone()], 0);
+    // Given
+    let sender_account_id = AccountId::from_str("sender@wonderland").expect("Valid");
+    let sender_keypair = KeyPair::generate().expect("Failed to generate sender KeyPair.");
+    let reciver_account_id = AccountId::from_str("reciver@wonderland").expect("Valid");
```

Comment:
```suggestion
    let receiver_account_id = AccountId::from_str("receiver@wonderland").expect("Valid");
```

That's why CI typo check might be usefull, however I can imagine how painful it would be to introduce in our project now